scripts/backfillKycEncryption.ts

```
@@ -0,0 +1,63 @@
+import { isEncrypted } from "../src/utils/piiEncryption";
+
+const APPLY_FLAG = "--apply";
+const applyChanges = process.argv.includes(APPLY_FLAG);
+
+async function main(): Promise<void> {
+  const records = await prisma.kycApplication.findMany({
+    where: {
+      OR: [
+        { machineRedactedPayload: { not: null } },
+        { machineExtractedPayload: { not: null } },
+      ],
```

ESLint

[error] Replace `⏎········{·machineRedactedPayload:·{·not:·null·}·},⏎········{·machineExtractedPayload:·{·not:·null·}·},⏎······` with `{·machineRedactedPayload:·{·not:·null·}·},·{·machineExtractedPayload:·{·not:·null·}·}`

(prettier/prettier)